spec/controllers/api/nonprofits_controller_spec.rb

```diff
@@ -27,7 +27,7 @@
 			input = {}
 			post '/api/nonprofits', params: input, xhr: true
 			expect(response).to have_http_status :unprocessable_entity
-			expect(response.parsed_body['errors'].keys).to match_array %w[name city state_code slug user_id]
+			expect(response.parsed_body['errors'].keys).to match_array %w[name city state_code slug user_id state_not_in_us]
```

Comment:
Once changing the errors in `app/models/nonprofit.rb` this change won't be needed.

Reply:
Indeed.